vpr/src/route/check_route.cpp

```diff
@@ -211,8 +211,8 @@ static void check_source(int inode, ClusterNetId net_id) {
                         "in check_source: net %d begins with a node of type %d.\n", size_t(net_id), rr_type);
     }
 
```

Comment:
Can we change the type of inode to RRNodeID and get rid of a bunch of the casting here?
Same for from_node and to_node.